src/pages/workspace/taxes/WorkspaceTaxesPage.tsx

```diff
@@ -0,0 +1,163 @@
+    );
+
+    const toggleTax = (tax: ListItem) => {
+        setSelectedTaxesIDs((prev) => {
+            if (prev.includes(tax.keyForList)) {
+                return prev.filter((item) => item !== tax.keyForList);
+            }
+            return [...prev, tax.keyForList];
+        });
+    };
+
+    const toggleAllTaxes = () => {
+        const taxesToSelect = taxesList.filter((tax) => tax.keyForList !== defaultExternalID && tax.keyForList !== foreignTaxDefault);
+        setSelectedTaxesIDs((prev) => {
+            if (prev.length === taxesToSelect.length) {
+                return [];
+            }
+
+            return taxesToSelect.map((item) => item.keyForList);
+        });
+    };
+
+    const getCustomListHeader = () => (
+        <View style={[styles.flex1, styles.flexRow, styles.justifyContentBetween, styles.pl3, styles.pr9]}>
+            <Text style={styles.searchInputStyle}>{translate('common.name')}</Text>
+            <Text style={[styles.searchInputStyle, styles.textAlignCenter]}>{translate('statusPage.status')}</Text>
+        </View>
+    );
+
+    const headerButtons = (
+        <View style={[styles.w100, styles.flexRow, isSmallScreenWidth && styles.mb3]}>
+            <Button
+                medium
+                success
+                onPress={() => {}}
+                icon={Expensicons.Plus}
+                text={translate('workspace.taxes.addRate')}
+                style={[styles.mr3, isSmallScreenWidth && styles.w50]}
+            />
+            <Button
+                medium
+                onPress={() => {}}
+                icon={Expensicons.Gear}
+                text={translate('common.settings')}
+                style={[isSmallScreenWidth && styles.w50]}
+            />
+        </View>
+    );
+
+    return (
+        <AdminPolicyAccessOrNotFoundWrapper policyID={policy?.id ?? ''}>
+            <PaidPolicyAccessOrNotFoundWrapper policyID={policy?.id ?? ''}>
+                <ScreenWrapper
+                    includeSafeAreaPaddingBottom={false}
+                    style={[styles.defaultModalContainer]}
+                    testID={WorkspaceTaxesPage.displayName}
+                    shouldShowOfflineIndicatorInWideScreen
+                >
+                    <HeaderWithBackButton
+                        icon={Illustrations.Coins}
+                        title={translate('workspace.common.taxes')}
+                        shouldShowBackButton={isSmallScreenWidth}
+                    >
+                        {!isSmallScreenWidth && headerButtons}
+                    </HeaderWithBackButton>
+
+                    {isSmallScreenWidth && <View style={[styles.pl5, styles.pr5]}>{headerButtons}</View>}
+
+                    <View style={[styles.ph5, styles.pb5]}>
+                        <Text style={[styles.textNormal, styles.colorMuted]}>{translate('workspace.taxes.subtitle')}</Text>
+                    </View>
+                    <SelectionList
```

Comment:
NAB for this PR since it's still a WIP feature, but we'll have to add a loading indicator later on similar to what was done here for [categories](https://github.com/Expensify/App/blob/ce878325a4d696c4cb0ec9f1496bba55ec8a815e/src/pages/workspace/categories/WorkspaceCategoriesPage.tsx#L173-L198) (without the empty page case since we always have defaults). We might not initially have this data locally when navigating to the page.

Reply:
Added!